src/app/tracking/tracking.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TrackingDirective } from './tracking.directive';
+
+describe('TrackingDirective', () => {
+  it('should create an instance', () => {
```

Comment:
I don't think this is a useful test. Can you test the compiled angular directive? We have an example here: https://github.com/AngularNYC/angular-presentation/blob/master/src/app/presentation/mode-routing/mode-routing.directive.spec.ts